std/range/package.d

```diff
@@ -5006,28 +5007,28 @@ struct Recurrence(alias fun, StateType, size_t stateSize)
 }
 
 ///
-@safe unittest
+pure @safe nothrow @nogc unittest
 {
     import std.algorithm.comparison : equal;
 
     // The Fibonacci numbers, using function in string form:
     // a[0] = 1, a[1] = 1, and compute a[n+1] = a[n-1] + a[n]
     auto fib = recurrence!("a[n-1] + a[n-2]")(1, 1);
-    assert(fib.take(10).equal([1, 1, 2, 3, 5, 8, 13, 21, 34, 55]));
+    assert(fib.take(10).equal(only(1, 1, 2, 3, 5, 8, 13, 21, 34, 55)));
```

Comment:
I don't think we should complicate ddoc-ed unit tests

Reply:
I can revert that, but this means I also have to take off the nogc attribute I added.

Reply:
Yes, I believe so. Making the ddoc-ed unit tests simpler is always better IMO. If you want to, you can always add a separate unit test block for the express purpose of testing `@nogc`, e.g. https://github.com/dlang/phobos/blob/9503a1e11ed9c474464b1ea6e5b07391407b18d3/std/range/package.d#L11862 (I would merge this even without that addition though).